content/applications/sales/point_of_sale/payment_methods/terminals/mercado_pago.rst

```diff
@@ -0,0 +1,101 @@
+============
+Mercado Pago
+============
+
+Connecting a payment terminal allows you to offer a fluid payment flow to your customers and ease
+the work of your cashiers.
+
+.. important::
+   - Odoo only supports **Point Smart** payment terminals. They can be purchased on
+     `Mercado Pago's website <https://www.mercadopago.com.mx/herramientas-para-vender/lectores-point>`_.
+   - Point Smart can only be used in the following :abbr:`LATAM (Latin America)` countries:
+
+     - Argentina
+     - Brazil
+     - Chile
+     - Colombia
+     - México
+     - Perú
+     - Uruguay
+
+.. seealso::
+   `Mercado Pago online payment <https://www.mercadopago.com.mx/herramientas-para-vender/check-out#benefits-checkout>`_
+
+Configuration
+=============
+
+#. Create your `Mercado Pago account <https://www.mercadopago.com.mx/>`_.
+#. Associate your terminal with a :guilabel:`store` and a :guilabel:`cash drawer` by following
+   `Mercado Pago's documentation <https://www.mercadopago.com.mx/developers/en/docs>`_.
+
+   .. note::
+      All the purchased terminals are automatically available on your Mercado dashboard.
+#. Set your Point Smart terminal to the :guilabel:`Point of Sale` operation mode.
+
+   .. warning::
+      Odoo does not support the Point Smart :guilabel:`Standalone` operation mode.
+#. :ref:`Generate your credentials <mercado_pago/credentials>`
+#. :ref:`Create and configure the related payment method <mercado_pago/method>`
+
+.. _mercado_pago/credentials:
+
+Generate your credentials
```

Comment:
```suggestion
Generate credentials
```
[No pronouns in headings](https://www.odoo.com/documentation/17.0/contributing/documentation/content_guidelines.html#titles-and-headings) :)